openfasoc/generators/glayout/glayout/flow/pdk/sky130_mapped/sky130_mapped.py

```diff
@@ -33,7 +33,7 @@
 openfasoc_dir = Path(__file__).resolve().parent.parent.parent.parent.parent.parent.parent
 
 klayout_drc_file = Path(__file__).resolve().parent / "sky130.lydrc"
-pdk_root = Path('/usr/bin/miniconda3/share/pdk/')
+pdk_root = Path('$PDK_ROOT')
```

Comment:
This has a lot of potential to fail. Please enclose all logic in a try and except block instead, where if the pdk_root can't be found, it defaults to the two possible locations (`/usr/bin/...` or `/home/$(logname)/...`)

Reply:
Can you check if it makes sense now. However it is expected that PDK_ROOT variable gets created during installation...as docker installation does it.